src/zgossip_engine.inc

```diff
@@ -83,6 +83,10 @@ typedef struct {
     size_t timeout;             //  Default client expiry timeout
     bool verbose;               //  Verbose logging enabled?
     char *log_prefix;           //  Default log prefix
+
```

Comment:
@wesyoung there's also the problem that all these changes are going to get lost when zgossip gets regenerated, as it's a zproto-generated engine. We can keep an eye on it and keep it there manually, but long term we should think of a proper solution

Reply:
bah- i've just started wrapping my head around zproto- that was an oversight. for some reason i was thinking the .inc wasn't auto-generated (i forget why), should have known better.

let me look through it a bit see what i can come up with. it doesn't make sense to keep it there manually (just creates confusion), if i can't find a decent path forward in the next few days- we can revert it and go from there.. thanks for catching this...

Reply:
No worries, and take your time. We don't regenerate that often (quite rarely actually) and the gossip message protocol is half broken anyway when regenerated.

So it's fine to keep it as-is for now